### PR TITLE
chore: added `npm outdated` command for `patch`,`minor`, and `major` commands

### LINKED ISSUE #145: chore: bump version scripts should check for outdated deps
Running the scripts `patch`, `minor`, and `major` should also run `outdated` so we get warned of publishing with outdated deps

### PR DESCRIPTION
**Pull request by Rishabhraghwendra18**
## chore: added `npm outdated` command for `patch`,`minor`, and `major` commands

Fixes #145 

Description of changes: 
Added `npm outdated` command in `package.json` file for `patch`, `minor`, and `major` commands.

Tag a reviewer: @ayoayco

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,9 +24,9 @@
     "lint:fix": "npm run lint:fix --workspaces --if-present",
     "build": "npm run build --workspaces --if-present",
     "test:watch": "npm run test:watch --workspaces --if-present",
-    "patch": "npm version patch -w",
-    "minor": "npm version minor -w",
-    "major": "npm version major -w",
+    "patch": "npm version patch -w && npm outdated",
```
Comment (ayoayco) [inline package.json:27]: I think outdated needs to go first because the version commands take an argument :)

**Review by ayoayco — CHANGES_REQUESTED** [review]
One change request!

File: package.json (hunk 1 above)
Comment (Rishabhraghwendra18) [reply package.json:27]: oh sorry I don't know about that

**Review by ayoayco — APPROVED** [review]
Thank you @Rishabhraghwendra18! Great update 🚀

**Pull request merged**